deployment/docker-swarm/external-db/README.md

```diff
@@ -0,0 +1,771 @@
+        secrets:
+            ALERT_ENCRYPTION_PASSWORD:
+              external: true
+              name: "<STACK_NAME>_ALERT_ENCRYPTION_PASSWORD"
+            
+    ```
+    - Replace <STACK_NAME> with the name of the stack to be used in the deployment.
+
+##### 2. Create ALERT_ENCRYPTION_GLOBAL_SALT secret.
+
+- Create a docker secret containing the encryption salt for Alert.
+    ```bash
+    docker secret create <STACK_NAME>_ALERT_ENCRYPTION_GLOBAL_SALT <FILE_CONTAINING_SALT>
+    ```
+    - Replace <STACK_NAME> with the name of the stack to be used in the deployment.
+    - Replace <FILE_CONTAINING_SALT> with the path to the file containing the salt text.
+    
+    Note: If you created the secret ALERT_ENCRYPTION_SALT in a version of Alert prior to 5.x, please use the same salt value for the ALERT_ENCRYPTION_GLOBAL_SALT secret.
+    
+- Make sure the alert service is uncommented from the docker-compose.local-overrides.yml file.
+- Uncomment the following from the docker-compose.local-overrides.yml file alert service section.
+    ```yaml
+        alert:
+            secrets:
+                - ALERT_ENCRYPTION_PASSWORD
+                - ALERT_ENCRYPTION_GLOBAL_SALT
+    ```
+- Uncomment the following from the secrets section of the docker-compose.local-overrides.yml file.
+    ```yaml
+        secrets:
+            ALERT_ENCRYPTION_PASSWORD:
+              external: true
+              name: "<STACK_NAME>_ALERT_ENCRYPTION_PASSWORD"
+            ALERT_ENCRYPTION_GLOBAL_SALT:
+              external: true
+              name: "<STACK_NAME>_ALERT_ENCRYPTION_GLOBAL_SALT"
+            
+    ```
+    - Replace <STACK_NAME> with the name of the stack to be used in the deployment.
+
+##### 3. Create ALERT_DB_USERNAME secret.
+
+- Create a docker secret containing the database username for Alert.
+    ```bash
+    docker secret create <STACK_NAME>ALERT_DB_USERNAME <FILE_CONTAINING_USER_NAME>
+    ```
+    - Replace <STACK_NAME> with the name of the stack to be used in the deployment.
+    - Replace <FILE_CONTAINING_USER_NAME> with the path to the file containing the database username.
+    
+- Make sure the alert service is uncommented from the docker-compose.local-overrides.yml file.
+- Uncomment the following secrets from the docker-compose.local-overrides.yml file alert service section.
+    ```yaml
+        alert:
+            secrets:
+              - ALERT_DB_USERNAME
+    ```
+- Uncomment the following from the secrets section of the docker-compose.local-overrides.yml file.
+    ```yaml
+        secrets:
+          ALERT_DB_USERNAME:
+            external: true
+            name: "<STACK_NAME>_ALERT_DB_USERNAME"
+            
+    ```
+    - Replace <STACK_NAME> with the name of the stack to be used in the deployment.
+
+##### 4. Create ALERT_DB_PASSWORD secret.
+
+- Create a docker secret containing the database password for Alert.
+    ```bash
+    docker secret create <STACK_NAME>ALERT_DB_PASSWORD <FILE_CONTAINING_PASSWORD>
```

Comment:
Should there be an `_` after `<STACK_NAME>`?

Reply:
Thank you